platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusSnapshot.java

```diff
@@ -131,57 +125,61 @@ public long nextConsensusNumber() {
     /**
      * Returns the minimum generation below which all events are ancient
      *
-     * @param roundsNonAncient
-     * 		the number of non-ancient rounds
+     * @param roundsNonAncient the number of non-ancient rounds
      * @return minimum non-ancient generation
      */
     public long getMinimumGenerationNonAncient(final int roundsNonAncient) {
```

Comment:
method name could be updated

Reply:
I don't think it's right to rename this method until the business logic inside it changes as well. I'm intentionally avoiding all changes to business logic in this PR.